### DISCUSSION TITLE
טעות בגליון 8 תרגיל 1 סעיף ב

### DISCUSSION THREAD

**Discussion by micha-sherman** [NuclearGandhi/technion_second_brain#5] [טעויות ותיקונים]
אני מאמין שאמור להיות שם sin במקום cos ושיש טעות העתקה

**Comment by micha-sherman** [comment]
גם בסעיף ח בגליון ניתן להגיע לתשובה סופית עבור המהירות הזוויתית על ידי שימור אנרגיה במקום  למדר מסדר שני

**Comment by micha-sherman** [comment]
ולדעתי בסעיף ט ניתן להגיע לתשובה על ידי משוואות תנע בכיוון e2' שים לב שהמתיחות מפעילה כוח רק בכיוון e1' ובגלל זה התאוצה בכיוון הנ"ל היא:
g*sin(teta)

**Comment by NuclearGandhi** [comment]
> אני מאמין שאמור להיות שם sin במקום cos ושיש טעות העתקה

יכל להיות, אבל כבר מחקתי את זה

> גם בסעיף ח בגליון ניתן להגיע לתשובה סופית עבור המהירות הזוויתית על ידי שימור אנרגיה במקום למדר מסדר שני

נכון חד משמעית עדיף, אשנה

>ולדעתי בסעיף ט ניתן להגיע לתשובה על ידי משוואות תנע בכיוון e2' שים לב שהמתיחות מפעילה כוח רק בכיוון e1' ובגלל זה התאוצה בכיוון הנ"ל היא:
> g*sin(teta)

וואלה כל השאלה הזאתי לא מוגדרת טוב, לא ברור לפי איזה משתנים הם רוצים שנציג את התשובה.

**Comment by micha-sherman** [comment]
כשמחקת את התשובות של א+ב מחקת גם את החישוב שמצדיק את סעיף ג'.
אתה צודק שאין שימור תנע כי אלה צירים שמשתנים בזמן אבל זה שציר e1'' קבוע במרחב לא מחייב את זה שיהיה שם שימור תנע.
זה תנאי הכרחי אבל לא מספיק.

**Reply by NuclearGandhi** [reply]
איייי, מחזיר
